src/data/automation.ts

```diff
@@ -116,7 +116,7 @@ export interface TimePatternTrigger extends BaseTrigger {
 
 export interface WebhookTrigger extends BaseTrigger {
   platform: "webhook";
-  webhook_id: string;
+  webhook_id?: string;
```

Comment:
Why did you make this change? This is not correct, webhook_id is required.

Reply:
I did this so I could use `undefined` as the initial state for `defaultConfig()`. That method is static, so there was no way to get access to the automation name from within. I've added a `DEFAULT_WEBHOOK_ID` in [ae2a58f](https://github.com/home-assistant/frontend/pull/11568/commits/ae2a58fe58fcbe63b461eaf535ce49a40273fd4a) instead of using undefined.